src/AsimovDeploy.WinAgent/Framework/Deployment/Steps/CopyPackageToTempFolder.cs

```diff
@@ -30,7 +30,7 @@ public CopyPackageToTempFolder(IAsimovConfig asimovConfig)
 
         public void Execute(DeployContext context)
         {
-            DirectoryUtil.CleanOldFiles(_asimovConfig.DownloadFolder);
+            DirectoryUtil.CleanOldFiles(_asimovConfig.DownloadFolder, 1);
```

Comment:
I would document what the 1 stands for (it's sufficient to give it a variable name)